core/drivers/fabric-driver/wallet-network1/admin.id

```diff
@@ -0,0 +1 @@
+{"credentials":{"certificate":"-----BEGIN CERTIFICATE-----\nMIIB9DCCAZugAwIBAgIURqnS2oBPZtH1JUMp8EDSrojstNYwCgYIKoZIzj0EAwIw\ncjELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMRowGAYDVQQK\nExFvcmcxLm5ldHdvcmsxLmNvbTEPMA0GA1UECxMGRmFicmljMR0wGwYDVQQDExRj\nYS5vcmcxLm5ldHdvcmsxLmNvbTAeFw0yMjExMDkwODU0MDBaFw0zMjExMDgyMDU5\nMDBaMCExDzANBgNVBAsTBmNsaWVudDEOMAwGA1UEAxMFYWRtaW4wWTATBgcqhkjO\nPQIBBggqhkjOPQMBBwNCAASEK5fIZImrOUqzB53juNU7Vzdfyg0PFOw6GjM2Aquo\nTvH2dpdRHwYRYtB5zQeq0d10kJheOJr89OPjhFaCMK+Xo2AwXjAOBgNVHQ8BAf8E\nBAMCB4AwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUOGVGgCHvpL9pvzqHwePsxmTd\n3f0wHwYDVR0jBBgwFoAUY5y3+yIO0VlnxzEpIIMazzOYW68wCgYIKoZIzj0EAwID\nRwAwRAIgBJC2RAjFtOS+3//xfJJplglkmCtxd/O008TJro1iH6YCIFWLaWxuA3Ce\na64vpCBgf90wXb0FJOTu/a5V8nsTGdey\n-----END CERTIFICATE-----\n","privateKey":"-----BEGIN PRIVATE KEY-----\r\nMIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQg5pfjc89eW8MRal9S\r\nxIIAK6plKeKCCWvvdQ89spa/iXuhRANCAASEK5fIZImrOUqzB53juNU7Vzdfyg0P\r\nFOw6GjM2AquoTvH2dpdRHwYRYtB5zQeq0d10kJheOJr89OPjhFaCMK+X\r\n-----END PRIVATE KEY-----\r\n"},"mspId":"Org1MSP","type":"X.509","version":1}
```

Comment:
Don't check in this file. This is generated on the fly when you are testing the driver.